sparql11/data-sparql11/manifest-all.ttl

```diff
@@ -11,6 +11,7 @@
 		<basic-update/manifest.ttl>
 		<bind/manifest.ttl>
 		<bindings/manifest.ttl>
+		<cast/manifest.ttl>
```

Comment:
There is a query specific manifest - could this added to `manifest-sparql-query.ttl` as well?

Reply:
Yes, sure. I didn't even realize those files existed, since it's also broken down in this file.